### DISCUSSION TITLE
Features visible to specific users/指定用户可见功能加入

### DISCUSSION THREAD

**Discussion by charmyang87** [usememos/memos#4157] [Ideas]
I hope this project can include a feature that allows content to be visible only to specific users. For example, when there are many users, some content, like private conversations between couples, should not be visible to the entire family.
希望这个项目能增加内容指定用户可见。   背景：当用户数较多时，有些内容只想给特定几个人看，比如夫妻间的交流，不希望全家都能看到。哈哈。

**Comment by RoccoSmit** [comment]
Adding reference to https://github.com/usememos/memos/issues/4111 as it has a similar theme

**Reply by charmyang87** [reply]
Yes, it's similar to what I need. Are there any plans to develop this function?
